axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/EventProcessorInfoConfiguration.java

```diff
@@ -39,7 +39,7 @@ public class EventProcessorInfoConfiguration implements Module {
 
 //    private final Component<EventProcessorControlService> eventProcessorControlService;
```

Comment:
I am skipping this file as it'll be rewritten as part of https://github.com/AxonFramework/AxonFramework/issues/3521 for 5.0.0